src/components/ec-metroline-item/ec-metroline-item.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-metroline-item ec-metroline-item--${id}` : `ec-metroline-item ec-metroline-item--${id}`"
+    class="ec-metroline-item"
+    :class="{
+      'ec-metroline-item--is-active': isActive || (isCompleted && isLast),
+      'ec-metroline-item--is-last': isLast
+    }"
+  >
+    <div
+      data-test="ec-metroline-item__status"
+      class="ec-metroline-item__status"
+    >
+      <div
+        data-test="ec-metroline-item__badge"
+        class="ec-metroline-item__badge"
+      >
+        <span
+          v-if="!isCompleted"
+          data-test="ec-metroline-item__index"
+        >
+          {{ badgeText || id.toString() }}
+        </span>
+
+        <ec-icon
+          v-else
+          data-test="ec-metroline-item__completed-icon"
+          name="simple-check"
+          :size="14"
+        />
+      </div>
+
+      <div
+        v-if="!isLast"
+        data-test="ec-metroline-item__status-bar"
+        class="ec-metroline-item__status-bar"
+      />
+    </div>
+
+    <div
+      data-test="ec-metroline-item__content"
+      class="ec-metroline-item__content"
+    >
+      <div
+        data-test="ec-metroline-item__header"
+        class="ec-metroline-item__header"
+      >
+        <div
+          data-test="ec-metroline-item__header-heading"
+          class="ec-metroline-item__header-heading"
+        >
+          <slot name="heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-sub-heading"
+          class="ec-metroline-item__header-sub-heading"
+        >
+          <slot name="sub-heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-cta"
+        >
+          <slot
+            v-if="!isReadOnly"
+            name="header-cta"
+            v-bind="{ goTo, goToNext, status, isLast, isReadOnly }"
+          />
+
+          <slot
+            v-else
+            name="header-cta-complete"
```

Comment:
@mcibique @matallo @jafcalvente I added this slot. After we complete the metroline we have an "Add description" cta. We want at the same time "edit" to don't be visible. We could hard code above and say 
```
    <div
          v-if="isCompleted && !isReadOnly"
          data-test="ec-metroline-item__header-cta"
        >
```

so this would hide the "edit" cta when metroline is complete. This though we also hide the field for "Add description"

We could also leave at the developer judgement to remember when we implement the metroline for each metro item to do

```
          <template #header-cta="{ goTo , isReadOnly }">
            <a href v-if="!isReadOnly" @click="goTo"> Edit </a>
            <a href="#" v-else> Add Description </a>
          </template>
```

But this also felt unnecessary work for the developer during implementation for each item and also bit risky in case he forgets you can complete a trade and go back to edit.

So i went for an after completion header slot, "header-cta-complete"

Let me know if you agree